config/settings.yml

```diff
@@ -355,6 +355,6 @@ discourse_ai:
     hidden: true
     type: list
 
-  ai_rag_pdf_images_enabled:
```

Comment:
Just noting here that a DB migration to clean up the site setting is not required since the site setting was only introduced 3 days ago.

Reply:
yeah not worth dealing with this imo, hidden setting, almost nobody changed it